sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -1107,7 +1130,7 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
     }
   }
 
-  test("SPARK-20364: Disable Parquet predicate pushdown for fields having dots in the names") {
+  test("SPARK-17636: Parquet predicate pushdown for fields having dots in the names") {
```

Comment:
@HyukjinKwon this PR also fixes the old problem that we don't allow to pushdown a column name containing "dot".